server/plugin/client_test.go

```diff
@@ -317,6 +317,80 @@ func TestCall(t *testing.T) {
 	}
 }
 
+func TestUpdatePipelineRunApprovalRequest(t *testing.T) {
+	defer monkey.UnpatchAll()
+	p := setupTestPlugin(&plugintest.API{})
+	for _, testCase := range []struct {
+		description string
+		err         error
+		statusCode  int
+	}{
+		{
+			description: "UpdatePipelineRunApprovalRequest: valid",
+			err:         nil,
+			statusCode:  http.StatusOK,
+		},
+		{
+			description: "UpdatePipelineRunApprovalRequest: with error",
+			err:         errors.New("mock-error"),
```

Comment:
change error and check everywhere

Reply:
Created a ticket for this. I am doing the same thing across multiple PRs. Once all the PRs are merged, I will create a separate PR for such small enhancements.
https://brightscout.atlassian.net/browse/MI-2511?atlOrigin=eyJpIjoiNWFlOTMxODJjYWM4NDgwYWJiNjE2N2MxMDdjZDRkNjAiLCJwIjoiaiJ9